lambdas/backend-api/src/templates/domain/user/user.ts

```diff
@@ -0,0 +1,3 @@
+export type User = {
+  id: string;
```

Comment:
Why is this `id` not `username` (or the other way around)? `type User` doesn't match `$User` - without the mapping you could do `schemaFor<User>` in the user-schema above too.

Reply:
I've removed User completely.